src/locale/locales/es/messages.po

```diff
@@ -1023,28 +1023,28 @@ msgstr "Libros"
 
 #: src/components/FeedInterstitials.tsx:350
 msgid "Browse more accounts on the Explore page"
-msgstr ""
+msgstr "Explorar más cuentas en la página Explore"
```

Comment:
Does the `Explore` page has a translation for the name? I think it should be used here instead of `página Explore`. Same where applicable.

Reply:
Sound odd the "explore the explore thing in the thingi exploring" xD
Should be a best way to solve this

Reply:
like... "Explorar más cuentas" ?